Corona-Warn-App/src/main/res/values/dimens.xml

```diff
@@ -145,6 +145,6 @@
     <!-- Contact Diary -->
     <dimen name="contact_diary_list_item">64dp</dimen>
     <dimen name="contact_diary_nested_recyclerview_margin">60dp</dimen>
-    <dimen name="statistics_card_height">328dp</dimen>
+    <dimen name="statistics_card_height">352dp</dimen>
```

Comment:
This is differs to the figma designs and uses too much whitespace when using regular font size. 

As discussed on Teams we should revert it and probably implement a dynamic card height until Friday.